src/api/companies/logout.js

```diff
@@ -0,0 +1,12 @@
+import { auth } from '../../firebase';
+
+export default function () {
+  return new Promise((resolve, reject) => {
+    auth.signOut()
+    .then((response) => {
+      resolve(response);
+    }).catch((error) => {
+      reject(error);
+    })
+  });
+}
```

Comment:
I have the same question than in the `companies/get.js`. Since the response is already a promise could we handle it like this:

```suggestion
export default function () {
  return auth.signOut();
}
```

Reply:
https://github.com/anonymouslyreportingtool/webapp/pull/5/commits/f4e970d561556d73bb936d4b502a8681685b105a